pkg/util/vpn.go

```diff
@@ -0,0 +1,129 @@
+package util
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type TailscaleOutput struct {
+	TailscaleIPs []string `json:"TailscaleIPs"`
+}
+
+// VPNInfo includes node information of the VPN. It is a general struct in case we want to add more vpn integrations
+type VPNInfo struct {
+	IPs    []net.IP
+	NodeID string
+}
+
+// vpnCliAuthInfo includes auth information of the VPN. It is a general struct in case we want to add more vpn integrations
+type vpnCliAuthInfo struct {
+	Name    string
+	JoinKey string
+}
+
+// StartVPN starts the VPN interface. General function in case we want to add more vpn integrations
+func StartVPN(vpnAuthConfigFile string) error {
+	authInfo, err := getVPNAuthInfo(vpnAuthConfigFile)
+	if err != nil {
+		return err
+	}
+
+	switch authInfo.Name {
+	case "tailscale":
+		outpt, err := execCommand("tailscale", []string{"up", "--authkey", authInfo.JoinKey, "--reset"})
+		if err != nil {
+			return err
+		}
+		logrus.Debugf("Output from tailscale up: %v", outpt)
+		return nil
+	default:
+		return fmt.Errorf("Requested VPN: %s is not supported. We currently only support tailscale", authInfo.Name)
```

Comment:
```suggestion
		return errors.New("Requested VPN: " + authInfo.Name + " is not supported. We currently only support tailscale")
```

Generally we should avoid using formatted strings with strings as it's a lot of unnecessary memory/heap usage due ot the use of reflection with no benefit.